tests/ui/attributes/mixed_export_name_and_no_mangle.rs

```diff
@@ -0,0 +1,10 @@
+// issue: rust-lang/rust#47446
+//@ run-rustfix
+//@ check-pass
+
```

Comment:
`compiletest` automatically [adds `-A unused`](https://github.com/rust-lang/rust/blob/1350eead10c46b9d3c2007fe0ea7892b7d7337ab/src/tools/compiletest/src/runtest.rs#L1603) for convenience, here we need to override this 
```suggestion

#![warn(unused_attributes)]

```

Reply:
Thanks a lot, it would have taken me like forever to figure that out.